Gemfile

```diff
@@ -1,10 +1,12 @@
 source 'https://rubygems.org'
 
-gem 'jekyll', '3.8.4'
+gem 'jekyll'
+gem 'kramdown-parser-gfm'
+gem 'rexml'
```

Comment:
Just noticing that GitHub pages uses Ruby v2 https://pages.github.com/versions/, so I might have to remove these changes.  Codespaces uses v3 by default, but we can switch it to v2.

Also noticed, we could use https://github.com/github/pages-gem to possibly simplify the gem file.

Ideally, we should move off Ruby and use React instead.